lib/promiscuous/new_relic/instrumentation.rb

```diff
@@ -1,5 +1,31 @@
 require 'new_relic/agent/instrumentation/controller_instrumentation'
 
+module PromiscuousNewRelicInstrumentedClass
+  extend ActiveSupport::Concern
+  include NewRelic::Agent::Instrumentation::ControllerInstrumentation
+
+  module ClassMethods
+    def instrument(method_name)
+      alias_method "orig_#{method_name}", method_name
+
+      define_method method_name do |*args, &block|
+        perform_action_with_newrelic_trace(:name => namespace_for_rpm(operation), :class_name => 'Subscriber', :force => true,
+                                           :category => "OtherTransaction/Promiscuous") do
+          __send__("orig_#{method_name}", *args, &block)
+        end
+      end
+    end
+  end
+
+
+  included do
+    private
+    def namespace_for_rpm(operation)
+      "#{self.app}/#{operation.model.to_s}/#{operation.operation}"
+    end
```

Comment:
not sure what we can do about that. this is totally separate from promiscuous itself, so there's no obligation for a promiscuous user to use this